app/experimenter/nimbus-ui/src/components/PageResults/TableVisualizationRow/index.stories.tsx

```diff
@@ -29,6 +29,7 @@ storiesOf("pages/Results/TableVisualizationRow", module)
       metricKey="retained"
       displayType={DISPLAY_TYPE.POPULATION}
       group={GROUP.OTHER}
+      isControlBranch
```

Comment:
I could have given this a default variable to make Storybook/maybe some future tests more convenient, but `isControlBranch` needs to be explicitely passed in everywhere we use it in the tables, so I'd rather keep it as required without a fallback.